app/src/experiments.py

```diff
@@ -85,7 +86,10 @@ def get_question(example: dict) -> str:
 def query_pipeline(example: dict) -> TaskOutput:
     question = get_question(example)
     logger.info("Getting answer for: %r", question)
-    response = create_pipeline().run_api(query=question)
+    kwargs = {"query": question}
```

Comment:
Make this less generic
```suggestion
   run_args = {"query": question}
```